source/write/bulk-write.txt

```diff
@@ -67,6 +67,11 @@ The following example creates an instance of ``InsertOne``:
 
 To insert multiple documents, create an instance of ``InsertOne`` for each document.
 
+.. note::
+
+   You must specify the ``_id`` field for each document you insert, otherwise the driver
+   throws a duplicate key error.
```

Comment:
Suggestion for clarity / avoiding [comma splice](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#commas)

```suggestion
   You must specify a value for the ``_id`` field for each document you insert. If you don't, the driver
   throws a duplicate key error.
```